config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   resources :books, only: [:index]
+  resources :authors, only: [:show]
+  get '/authors/:id/books', to: 'authors#books'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.